.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: ruby
 cache: bundler
 rvm:
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-9.0.0.0
+  - jruby-9.0.4.0
```

Comment:
Yes but it worked :) 

I'll remove 9.0.4.0, if it works on 9.0.0.0 it should work on the newer version too.